src/odemis/acq/test/move_test.py

```diff
@@ -71,75 +73,101 @@ def setUp(self):
-        f = cryoLoadSample(LOADING)  # Loading first, then Imaging
+        # Test tilting from imaging
+        # Get the stage to imaging position
+        f = cryoLoadSample(LOADING)
         f.result()
         f = cryoLoadSample(IMAGING)
         f.result()
-        f = focus.moveAbs(self.focus_deactive)
-        f.result()
-        f = stage.moveAbs({'rx': 0.003, 'rz': 0.003})
+
+        # Tilt the stage
+        f = cryoTiltSample(rx=self.tilt_angle, rz=self.tilt_angle)
         f.result()
+        test.assert_pos_almost_equal(stage.position.value, {'rx': self.tilt_angle, 'rz': self.tilt_angle},
+                                     match_all=False,
+                                     atol=ATOL_ROTATION_POS)
+
+        # Test imaging from tilting
         f = cryoTiltSample(rx=0, rz=0)
         f.result()
-        test.assert_pos_almost_equal(stage.position.value, {'rx': 0, 'rz': 0}, match_all=False,
-                                     atol=ATOL_STAGE)
+        test.assert_pos_almost_equal(stage.position.value, self.stage_active, atol=ATOL_LINEAR_POS)
 
-    def test_tilting_from_imaging(self):
+    def test_invalid_switch_movements(self):
         """
-        Test tilting the sample stage while the stage is in Imaging position
+        Test it's not possible to do some disallowed switch movements
         """
-        stage = self.stage
-        focus = self.focus
-        # Get the stage to imaging position and park focus
+        # Test tilting from loading
         f = cryoLoadSample(LOADING)
         f.result()
-        f = cryoLoadSample(IMAGING)
-        f.result()
-        f = focus.moveAbs(self.focus_deactive)
+        with self.assertRaises(ValueError):
+            f = cryoTiltSample(rx=self.tilt_angle, rz=self.tilt_angle)
+            f.result()
+
+        # Test coating from imaging
+        f = cryoLoadSample(LOADING)
         f.result()
-        # Tilt the stage
-        f = cryoTiltSample(rx=0.003, rz=0.003)
+        f = cryoLoadSample(IMAGING)
         f.result()
-        test.assert_pos_almost_equal(stage.position.value, {'rx': 0.003, 'rz': 0.003}, match_all=False,
-                                     atol=ATOL_STAGE)
+        with self.assertRaises(ValueError):
+            f = cryoLoadSample(COATING)
+            f.result()
 
-    def test_tilting_from_loading(self):
-        """
-        Test it's not possible to do tilting movement while the stage is in Loading position
-        """
-        stage = self.stage
+        # Test imaging from coating
         f = cryoLoadSample(LOADING)
         f.result()
+        f = cryoLoadSample(COATING)
+        f.result()
+        with self.assertRaises(ValueError):
+            f = cryoLoadSample(IMAGING)
+            f.result()
+
+        # Test tilting from coating
         with self.assertRaises(ValueError):
-            f = cryoTiltSample(rx=0.003, rz=0.003)
+            f = cryoTiltSample(rx=self.tilt_angle, rz=self.tilt_angle)
```

Comment:
It doesn't really make sense to use the same angle for both, as they are unrelated. So at least, use different variable names.
Also, please test with rz=0, it's a corner-case which is going to be very common.